app/home/homeController.js

```diff
@@ -8,7 +8,9 @@
 
     function controllerFunction (cityService) {
           var vm = this;
-          vm.city = cityService.city;
+          vm.cityService = cityService;
+          // Curious as to why you need the whole object. Commented out in forecastController.
```

Comment:
Let me clarify, both controllers have access to the city attribute. But upon changing it in the home controller via the input it does not persist to the forecast controller.